shotgun_api3/shotgun.py

```diff
@@ -1573,6 +1608,13 @@ def _auth_params(self):
                 "script_key" : str(self.config.api_key),
             }
 
+        # Authenticate using session_id
+        elif self.config.session_token:
+            auth_params = {
+                "session_token" : str(self.config.session_token),
+                "reject_if_expired": False
```

Comment:
Oops, some leftovers from testing. Thanks for spotting this. Shouldn't be in there.